images/router/haproxy/reload-haproxy

```diff
@@ -15,7 +16,37 @@ if [ -f $pid_file ]; then
 fi
 
 if [ -n "$old_pid" ]; then
+  if $(set | grep DROP_SYN_DURING_RESTART= > /dev/null) && [ -n "$DROP_SYN_DURING_RESTART" ]; then
+    # We install the syn eater so that connections that come in during the restart don't
+    # go onto the wrong socket, which is then closed.  The sleep is needed to let the socket
+    # drain before the new haproxy starts and binds to the same port.
+    ports=$(grep 'bind :' $config_file | cut -f2 -d: | paste -d, -s)
```

Comment:
Maybe a lil' more stricter expression would be better here for `bind :` part  ala: `grep -E 'bind\s+:'`  or `egrep ...` or even something like: `sed 's#\s*bind.*:\([0-9]\+\).*#\1#g'`.